bin/mail/2016-05-02-petition-followup/generate-list.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# All US based teachers and HOC organizers who have not clicked through the previous petition email.
+
+# First, get all US teachers and past organizers.
+teacher_query = %q(
+  ((kind_s:user && role_s:teacher) ||
+  kind_s:HocSignup2015 || kind_s:HocSignup2014 || kind_s:"CSEdWeekEvent2013") && (
```

Comment:
nit: Is there a reason for the quotes around CSEdWeekEvent2013 but not the others?